en/ros2/px4_ros2_control_interface.md

```diff
@@ -0,0 +1,455 @@
+# PX4 ROS 2 Control Interface <Badge type="warning" text="main (v1.15+)" vertical="top" />
```

Comment:
FYI @GuillaumeLaine I just added this `Badge` to the headings in the three docs. Part of a plan to make it clear the versions in which stuff is known to work.

Note I'd rather do this using page front metadata, but that's going to require some more thinking.